Add docs for new validation features in .NET 11 Preview 5 (#37235)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/localization/make-content-localizable.md

In the preceding code, `SharedResource` is the class corresponding to the *.resx* file where the validation messages are stored. With this approach, DataAnnotations only uses `SharedResource`, rather than the resource for each class.

:::moniker range=">= aspnetcore-11.0"

## DataAnnotations localization in Minimal APIs and Blazor

Validation localization is available for Minimal API and Blazor apps that opt into the `Microsoft.Extensions.Validation` pipeline by calling `AddValidation()` in `Program.cs`. Localize validation error messages and the display names of validated properties and parameters by also calling `AddValidationLocalization`:

```csharp
builder.Services.AddValidation();

> builder.Services.AddValidationLocalization<ValidationResources>();

Author: AddValidationLocalization already calls AddLocalization. Customization shown in later example.

```

The localization integration does not apply to MVC and Razor Pages apps, or to Blazor forms that don't include `AddValidation`.

> [!NOTE]

> > The integration is provided by the `Microsoft.Extensions.Validation.Localization` package, which builds on the `Microsoft.Extensions.Validation` package. Both packages are included in the Web SDK (`Microsoft.NET.Sdk.Web`) and the Razor SDK (`Microsoft.NET.Sdk.Razor`), so ASP.NET Core and Blazor apps don't need explicit package references. Standalone Blazor WebAssembly apps and class libraries based on `Microsoft.NET.Sdk` reference both packages explicitly:

Author: ```suggestion
 > The integration is provided by the `Microsoft.Extensions.Validation.Localization` package, which builds on the `Microsoft.Extensions.Validation` package. Both packages are included in the Web SDK (`Microsoft.NET.Sdk.Web`) and the Razor SDK (`Microsoft.NET.Sdk.Razor`), so apps that use those SDKs don't need explicit package references. Standalone Blazor WebAssembly apps and other project that do not use the Web SDK or the Razor SDK must reference both packages explicitly:
```
